Resources/Prototypes/Loadouts/head.yml

```diff
@@ -272,3 +288,86 @@
   exclusive: true
   items:
     - ClothingHeadBandBrown
+
+- type: loadout
+  id: LoadoutHeadFishCap
+  category: Head
+  cost: 2
+  exclusive: true
+  items:
+    - ClothingHeadFishCap
+
+- type: loadout
+  id: LoadoutHeadRastaHat
+  category: Head
+  cost: 4
+  exclusive: true
+  items:
+    - ClothingHeadRastaHat
+
+# Flatcaps
+- type: loadout
+  id: LoadoutHeadGreyFlatcap
+  category: Head
+  cost: 2
+  exclusive: true
+  items:
+    - ClothingHeadHatGreyFlatcap
+
+- type: loadout
+  id: LoadoutHeadBrownFlatcap
+  category: Head
+  cost: 2
+  exclusive: true
+  items:
+    - ClothingHeadHatBrownFlatcap
```

Comment:
Head items are also probably going to need restricted to Security by "Security Color Scheme Only". They look really cool with Bandanas, Hats, etc. But if the hat doesn't match their color scheme, it's going to easily break the visibility rule. 

One that doesn't work
![image](https://github.com/user-attachments/assets/a05738b5-0e58-4588-9fdd-5ec0e3213ad7)

One that does
![image](https://github.com/user-attachments/assets/7f29d2d8-a07e-4068-89cb-c69cec34733f)

Reply:
Also I'd like to say, holy shit "Secoff in a black cowboy hat" is extremely fucking cool.

![image](https://github.com/user-attachments/assets/6de86b30-d140-41e0-be6f-3cbc47e77380)

Reply:
Wouldn't you already be able to make such outlandish colors anyway with your hair? These randomly-colored hair can even extend to below the shoulder, while the hats only stay on top of your head. 

Examples:
![image](https://github.com/user-attachments/assets/0a141d1d-2518-4691-838e-ce89cc938670)
![image](https://github.com/user-attachments/assets/49bab3cf-05e6-4cd7-b729-27e13ef1b184)

Reply:
Guh, not really anything we can do about the hair system. But let's keep it clean where we can. :)

Reply:
Resolved with [38b5605](https://github.com/Simple-Station/Einstein-Engines/pull/616/commits/38b560569fbb51acc15468ee7532d4f51c7f5623)